<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorLightWidget</class>
 <widget class="QWidget" name="InspectorLightWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1,1,1,1,0">
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Light source type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ambient color</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,1,0,1,0,1,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="rLabel">
        <property name="text">
         <string>r</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="rSpin">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000001000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gLabel">
        <property name="text">
         <string>g</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="gSpin">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000001000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bLabel">
        <property name="text">
         <string>b</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="bSpin">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000001000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="aLabel">
        <property name="text">
         <string>a</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="aSpin">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000001000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Diffuse color</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,1,0,1,0,1,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="rLabel_2">
        <property name="text">
         <string>r</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="rSpin_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000001000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gLabel_2">
        <property name="text">
         <string>g</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="gSpin_2">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000001000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bLabel_2">
        <property name="text">
         <string>b</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="bSpin_2">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000001000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="aLabel_2">
        <property name="text">
         <string>a</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="aSpin_2">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000001000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Specular color</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0,1,0,1,0,1,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="rLabel_3">
        <property name="text">
         <string>r</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="rSpin_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000001000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gLabel_3">
        <property name="text">
         <string>g</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="gSpin_3">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000001000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bLabel_3">
        <property name="text">
         <string>b</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="bSpin_3">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000001000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="aLabel_3">
        <property name="text">
         <string>a</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="aSpin_3">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000001000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Spotlight direction</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1,0,1,0,1,0,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="xLabel">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="xSpin">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="yLabel">
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ySpin">
        <property name="decimals">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="zLabel">
        <property name="text">
         <string>z</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="zSpin">
        <property name="decimals">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="wLabel">
        <property name="text">
         <string>w</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="wSpin">
        <property name="decimals">
         <number>6</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="appBtn">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
